### PR TITLE
created google react chart

### PR DESCRIPTION
**Pull request by DustinHaefele**
Added a google chart to display total contributions

### CODE REVIEW

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export default class Dashboard extends React.Component {
       <section className='dashboard'>
         <header>Dashboard</header>
         {myData}
+        <TotalContributions contribs={{total_donations: 10000, spent: 9000}} />
```
Comment (lucasvocos) [inline src/components/Dashboard/Dashboard.js]: This is just a placeholder right?

File: src/components/TotalContributions/TotalContributions.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+import ContribsChart from './ContribsChart';
 
 function TotalContributions(props){
 
   return (
     <div>
-      <h3>total donations in most recent cycle: ${props.contribs.total_donations.toLocaleString()}</h3>
-      <h3>spent in most recent cycle: ${props.contribs.spent.toLocaleString()}</h3>
-      <h3>Cash that the campaign has on hand: ${props.contribs.cash_on_hand.toLocaleString()}</h3>
+      <ContribsChart donations={props.contribs.total_donations} spent={props.contribs.spent}/>
```
Comment (lucasvocos) [inline src/components/TotalContributions/TotalContributions.js]: Where is props.contribs.spent coming from

File: src/components/Dashboard/Dashboard.js (hunk 1 above)
Comment (DustinHaefele) [reply src/components/Dashboard/Dashboard.js]: Sorry yeah that is a placeholder I forgot to delete... Will fix it and push back up

File: src/components/TotalContributions/TotalContributions.js (hunk 2 above)
Comment (DustinHaefele) [reply src/components/TotalContributions/TotalContributions.js]: Should be coming from the individual rep component

**Pull request merged**